pkgs/tools/video/ffsubsync/default.nix

```
@@ -0,0 +1,45 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "ffsubsync";
+  version = "0.4.25";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "smacke";
+    repo = pname;
```

```suggestion
    repo = pname;
    repo = "ffsubsync";
```

Avoid using `pname` here